src/wui/minimap.cc

```diff
@@ -235,18 +237,20 @@ void MiniMap::toggle(MiniMapLayer const button) {
 
 void MiniMap::resize() {
 	view_.set_zoom(*view_.minimap_layers_ & MiniMapLayer::Zoom2);
-	set_inner_size(view_.get_w(), view_.get_h() + number_of_button_rows() * but_h());
+	// Read number of rows after the zoom.
+	const auto rows = number_of_button_rows();
```

Comment:
```suggestion
	const uint32_t rows = number_of_button_rows();
```
No `auto` for primitive types please for readability